roles/nova/tasks/main.yml

```diff
@@ -40,6 +40,17 @@
   ansible.builtin.import_tasks:
     file: generate_public_key.yml
 
+# NOTE(okozachenko1203): We cannot set helm hook for this bootstrap job.
+# As a workaround, remove this job before upgrade nova helm release.
+- name: Remove nova-bootstrap job
+  failed_when: false
```

Comment:
I think we might need to add changed_when: false too so we can avoid the idempotence issues.